documentation/identifiers.md

```diff
@@ -78,65 +88,94 @@ There are many different kinds of organisations that give, receive or benefit fr
 
-2. **Registered charity number (UK)**
+In these cases, it's important to know which identifier to pick so that users of data have the best possible chance of understanding that two grants have been made to the same organisation. 
 
-   If you have a **registered charity number** for the organisation, use the Registration Agency Code:
-    * 'GB-CHC-' for a [charity registered in England and Wales](http://www.charitycommission.gov.uk/), 
-    * 'GB-SC-' for a [charity registered in Scotland](http://www.oscr.org.uk/), or
-    * 'GB-NIC-' for a [charity registered in Northern Ireland](http://www.charitycommissionni.org.uk/) 
-    
-    along with the registered number.
+[org-id.guide](http://org-id.guide) ranks identifier lists by relevance and quality to help you pick the best identifier. 
 
-3.  **Educational establishment (UK)**
+```eval_rst
 
-    If you have an educational establishment for the organisation, use the Registration Agency Code:
-      * 'GB-EDU-' for a school, university or other educational establishment in England and Wales along with the [EduBase](http://www.education.gov.uk/edubase/home.xhtml) URN identifier. 
-      * 'GB-UKPRN-' for a school, university or other educational establishment in Scotland or Northern Ireland along with the [UK Register of Learning](https://www.ukrlp.co.uk/) UKPRN number.
+.. hint:: Relevance and quality defined: 
 
-4. **Other registered number**
+  * Relevance: are you likely to find the organisation you are looking for in this list?
+  * Quality: are the identifiers in this list stable and linked to open, accessible contextual data, and can they be easily mapped to other identifiers.
+```
 
-    If you have a registered number from some other scheme, including overseas registrars, check the [IATI Organisation Registration Agency Codelist](http://iatistandard.org/codelists/OrganisationRegistrationAgency/) for a Registration Agency Code to use. If the Registration Agency Code you need is not listed, [contact the support team](http://www.threesixtygiving.org/contact/).
+Search on [org-id.guide](http://org-id.guide) for identifier sources for [UK organisations](http://org-id.guide/?structure=&coverage=GB&subnational=&sector=), [UK charities](http://org-id.guide/?structure=charity&coverage=GB&sector=), or [any other organisation type](http://org-id.guide/).
 
-5. **No registered number**
+### Commonly used identifier lists
 
-    If you do not have any external registration numbers for the organisation, use your 360Giving prefix and any internal identifier* you have for this organisation.
-    
-    *If you use a database that records details of organisations in a separate lookup table, this may provide an identifier you can use. 
+The following identifier lists are often used in 360Giving publication. They are listed here in rough order of priority (e.g. if you already know the company number, use this in preference to the charity number).
```

Comment:
Actually, and this came up also in our board meeting, we prefer to give priority to the charity number, not the company number. This should be changed. I do think we need to guide publishers to record both company house and charity numbers in the schema, and choose the charity number as the main identifier.

Reply:
I think we may need to build a tool that helps them automatically find the company number based on the charity number. Its unlikely they'll collect company numbers and very few will want to add this unless we make it very easy for them to.

Reply:
I don't think we have to ask them to add them. If they have charity numbers - that's good. It's up the intermediaries to do the mapping.

Reply:
I suggest removing this sentence: "They are listed here in rough order of priority (e.g. if you already know the company number, use this in preference to the charity number)."